CancelBooking UseCase/CancelBookinng.java

```diff
@@ -146,7 +152,9 @@ public void showCancellationPolicy() {
 }
 // Presentation Layer
 // Main class to handle user interaction for making and canceling reservations
-public class CancelBooking {
+
+
```

Comment:
```suggestion
instead of dealing with logic and UI together in the "main" method, you can separate concerns into distinct methods and keep "main" only for managing control flow. also, you can add input validation for room number and CPR input to make sure they are not empty or invalid.
```